### PR TITLE
Kindly accept my PR

### PR DESCRIPTION
**Pull request by harsh3311**
I have addressed some of the issues such as:

Changed the import statement for React.
Added a Link component from react-router-dom.
Fixed the onClick handlers to Link components.
Fixed the class attribute to className.
Fixed the style attributes to accept JavaScript objects.

 All Submissions:

* [ ] Have you followed the guidelines in our Contributing document?
* [ ] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?

### New Feature Submissions:

1. [ ] Does your submission pass tests?
2. [ ] Have you lint your code locally before submission?

### Changes to Core Features:

* [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
* [ ] Have you written new tests for your core changes, as applicable?
* [ ] Have you successfully run tests with your changes locally?

### CODE REVIEW

**Comment by harsh3311** [comment]
Hi Sir, kindly see my PR

File: src/Component/Themes/DropToSelection.js [hunk 1]
```diff
@@ -168,32 +168,32 @@ class CircleWithInfoAnimations extends React.Component {
...
 
     allowDrop(ev) {
-        //alert(JSON.stringify(ev.target.id))
+        alert(JSON.stringify(ev.target.id))
         ev.preventDefault();
     }
 
 
     drag(ev) {
-        //alert(JSON.stringify(ev.target.id))
```
Comment (anjula-sack) [inline src/Component/Themes/DropToSelection.js:189]: You are suppose to remove the comments not uncomment them @harsh3311

**Review by anjula-sack — CHANGES_REQUESTED** [review]

File: src/Component/Themes/DropToSelection.js (hunk 1 above)
Comment (harsh3311) [reply src/Component/Themes/DropToSelection.js:189]: Sure i will do so

Comment (anjula-sack) [reply src/Component/Themes/DropToSelection.js:189]: Any updates? @harsh3311

**Pull request closed without merging**